stacks-core/Cargo.toml

```diff
@@ -27,6 +27,7 @@ serde = { workspace = true, features = ["derive"] }
 sha2.workspace = true
 strum = { workspace = true, features = ["derive"] }
 thiserror.workspace = true
+anyhow.workspace = true
```

Comment:
Did you forget to remove the dependency?

Reply:
`anyhow` is being used in unit tests. Lmk if there's a preferred way to handle results in those cases.